projects/go-lib/src/lib/components/go-toast/go-toast.component.html

```diff
@@ -22,13 +23,14 @@
     </h4>
 
     <ng-container *ngIf="messageContent; else messageContentElse">
-      <p class="go-toast-content__message">
+      <p class="go-toast-content__message" [ngClass]="{'go-toast-content--dark': theme === 'dark'}">
        <ng-container *ngTemplateOutlet="messageContent"></ng-container>
       </p>
     </ng-container>
     <ng-template #messageContentElse>
       <p
         class="go-toast-content__message"
+        [ngClass]="{'go-toast-content--dark': theme === 'dark'}"
```

Comment:
It seems like we can probably get rid of these 3 `ngClass` checks in favor of using the CSS cascade. These `ngClass`es are going to re-evaluate with every lifecycle. It will be slightly more performant to handle this in CSS.

See my comments in the scss file.

Reply:
This can be removed.